src/fio/fioInfo.js

```diff
@@ -7,25 +7,26 @@ import { imageServerUrl } from '../common/utils'
 
 const defaultSettings: any = {
   apiUrls: [
-    'https://fio.greymass.com:443/v1/',
-    'https://fio.zenblocks.io:443/v1/',
-    'https://api.fio.alohaeos.com:443/v1/',
-    'https://fio-mainnet.eosblocksmith.io:443/v1/',
-    'https://fio.eu.eosamsterdam.net:443/v1/',
-    'https://fio.eosdac.io:443/v1/',
+    'https://fio.eu.eosamsterdam.net/v1/',
+    'https://fio.eosdac.io/v1/',
     'http://fioapi.nodeone.io:6881/v1/',
-    'https://fio.acherontrading.com:443/v1/',
-    'https://fio.eos.barcelona:443/v1/',
-    'https://fio.eosusa.news:443/v1/',
-    'https://fio.eosargentina.io:443/v1/',
-    'https://api.fio.currencyhub.io:443/v1/',
-    'https://fio.eoscannon.io:443/v1/',
-    'https://fio.eossweden.org:443/v1/',
-    'https://fio.maltablock.org:443/v1/',
-    'https://api.fio.eosdetroit.io:443/v1/',
-    'https://fio.eosdublin.io:443/v1/',
-    'https://fio.eosphere.io:443/v1/',
-    'https://fio.cryptolions.io:443/v1/'
+    'https://fio.eosphere.io/v1/',
```

Comment:
Is this node live?

Reply:
Confirmed this node is valid and working.